scripts/publish_binary_size.js

```diff
@@ -33,13 +33,16 @@ github.authenticate({type: 'app', token});
 const platforms = [
     { 'platform': 'iOS', 'arch': 'armv7' },
     { 'platform': 'iOS', 'arch': 'arm64' },
+    { 'platform': 'iOS', 'arch': 'Dynamic' },
```

Comment:
Note that their pair PR to keep these constants in sync is here: https://github.com/mapbox/mapbox-gl-native/pull/13876